index.js

```diff
@@ -14,14 +14,43 @@ module.exports = opts => buf => {
 		return Promise.resolve(buf);
 	}
 
+	// Default is to strip all markers from output file
+	opts.stripCom = typeof opts.stripCom === 'undefined' ? true : opts.stripCom;
+	opts.stripExif = typeof opts.stripExif === 'undefined' ? true : opts.stripExif;
+	opts.stripIptc = typeof opts.stripIptc === 'undefined' ? true : opts.stripIptc;
+	opts.stripIcc = typeof opts.stripIcc === 'undefined' ? true : opts.stripIcc;
+	opts.stripXmp = typeof opts.stripXmp === 'undefined' ? true : opts.stripXmp;
```

Comment:
Do this at the top where the `opts` assignment is instead:

```js
opts = Object.assign({
    stripAll: true,
    stripIptc: true,
    stripIcc: true
}, opts);
```

To set individual markers, you'd set `stripAll` to `false` and then enable the ones you want.